worker/runtime/integration/integration_test.go

```diff
@@ -38,10 +38,16 @@ func (s *IntegrationSuite) containerdSocket() string {
 }
 
 func (s *IntegrationSuite) startContainerd() {
+	configPath := filepath.Join(s.tmpDir, "containerd.toml")
+	config := `oom_score = -999`
```

Comment:
Thoughts on exposing `writeDefaultContainerdConfig` and using that here so that we're running the integration tests with the same (default) config that Concourse runs with? I guess we're running testflight against the default config, but can't hurt to keep the integration tests in sync too

Reply:
@aoldershaw I like the idea of having the same default config. It seems safest to disable the same plugins we're disabling in production.

Trying to think of a neat way of exposing `writeDefaultContainerdConfig` as everything in that file is tied up with `WorkerCmd`, and we wouldn't want to put it in there. Do you think it makes sense to introduce a new `ContainerdRunner` interface in the `containerd.go` file, and expose everything through that? Would need a little bit of refactoring

The other option is to duplicate `writeDefaultContainerdConfig` in the test. The con being it would go out of sync when the default config gets updated.